README.md

```diff
@@ -76,15 +76,15 @@ $ git clone --recurse-submodules https://github.com/rapidsai/rmm.git
 $ cd rmm
 ```
 
-Follow the instructions under "Create the conda development environment `cudf_dev`" in the
+Follow the instructions under "Create the conda development environment `rmm_dev`" in the
 [cuDF README](https://github.com/rapidsai/cudf#build-from-source).
 
-- Create the conda development environment `cudf_dev`
+- Create the conda development environment `rmm_dev`
 ```bash
 # create the conda environment (assuming in base `cudf` directory)
```

Comment:
Wait, why point them to the cuDF instructions if you just repeat them here with RMM naming? Do one or the other.